Resources/Prototypes/Traits/disabilities.yml

```
@@ -14,7 +14,7 @@
         - Photophobia
         - Nearsighted
         - CyberEyes
-        - FlareShieldingModule
+      # - FlareShieldingModule
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)